Dockerfile

```diff
@@ -9,8 +9,8 @@ FROM ${IMAGE}
 
 RUN mkdir -p /app /app/logs /app/tmp
 
-ARG SOPS_AMD64="https://github.com/bekk/sops/releases/download/v3/sops-v3.linux.amd64"
-ARG SOPS_ARM64="https://github.com/bekk/sops/releases/download/v3/sops-v3.linux.arm64"
+ARG SOPS_AMD64="https://github.com/bekk/sops/releases/download/v3.9.4/sops-v3.linux.amd64"
+ARG SOPS_ARM64="https://github.com/bekk/sops/releases/download/v3.9.4/sops-v3.linux.arm64"
```

Comment:
Det overrasker meg litt at dette er en bump. 
Vanligvis ville dette vært pinning av versjon for å unngå påfølgende versjoner.
 Hvilken versjon fikk du fra kun v3?

Reply:
Det er kanskje ikke versjonering på samme måte. Du sikter vel bare på en tag https://github.com/bekk/sops/tags

Reply:
Bare resolve